motr/io_req.c

```diff
@@ -1230,8 +1226,8 @@ static bool verify_checksum(struct m0_op_io *ioo)
 	}
 
 	if (m0_bufvec_cursor_move(&datacur, 0) &&
-	    m0_ivec_cursor_move(&extcur, 0) &&
-	    attr_idx == ioo->ioo_attr.ov_vec.v_nr) {
+	    m0_ivec_cursor_move(&extcur, 0)) {
+	    //attr_idx == ioo->ioo_attr.ov_vec.v_nr) {
```

Comment:
Commented code if not needed, can be removed.

Reply:
done